### PR TITLE
add ubuntu:lunar builder target and test it daily

### PR DESCRIPTION
**Pull request by Habbie**
### Checklist

I have:
- [x] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [x] compiled this code
- [x] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [x] added or modified regression test(s)
- [ ] added or modified unit test(s)
- [ ]  checked that this code was merged to master

### CODE REVIEW

**Comment by Habbie** [comment]
successfully built in https://github.com/Habbie/pdns/actions/runs/4356597061/jobs/7614830176

File: builder-support/dockerfiles/Dockerfile.target.ubuntu-lunar-ard64 [hunk 1]
```diff
@@ -0,0 +1 @@
+Dockerfile.target.ubuntu-lunar
```
Comment (rgacogne) [inline builder-support/dockerfiles/Dockerfile.target.ubuntu-lunar-ard64:1]: s/Dockerfile.target.ubuntu-lunar-ard64/Dockerfile.target.ubuntu-lunar-arm64/ ?

Comment (Habbie) [reply builder-support/dockerfiles/Dockerfile.target.ubuntu-lunar-ard64:1]: oh wow. thanks. :)

**Review by rgacogne — APPROVED** [review]
Looks good, assuming that you actually completely removed `builder-support/dockerfiles/Dockerfile.target.ubuntu-lunar-arm64` on purpose :)

**Review by rgacogne — APPROVED** [review]

**Pull request merged**